### PR TITLE
Copyedit transcribe handwritten text with python and azure

### PR DESCRIPTION
**Pull request by charlottejmc**
I am preparing this pull request to apply my copyedits for EN original lesson transcribe-handwritten-text-with-python-and-azure.

### CODE REVIEW

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: In dialogue with Alex, I have split up “Procedure” into a “Setting Up” section and a “Projects” section. I thought this gave a clearer indication to the reader of what they could expect from each section. It also meant there were fewer embedded sections in the previous "6. Transcribe Handwriting" section, which allows us to avoid roman numerals.

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: To improve clarity, I also suggest alphabetising rather than numbering the sub-steps. This helps differentiate them from the main steps. I changed any mention to other “sections” or “steps” to account for these changes. Please let me know if this makes sense to you!

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: I have changed section titles to -ing action verbs. This helps to engage readers and fits in with the PH house style developed across the lessons so far.

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: I found most of the endnotes functioned more effectively as embedded links within the text.

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md:152]: In dialogue with Alex, I have removed references to line numbers that appear within your code blocks throughout the lesson. It is unclear which lines these refer to, and your explanations are already so detailed that the reader should be able to navigate around the code themselves!

File: en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md
Comment (charlottejmc) [inline en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: I am also suggesting changing the lesson title to "Transcribing" rather than "Transcribe" to fit with the PH house style across our lesson directory, where we use action verbs. I will update the image filenames and associated directory names accordingly.

Comment (hawc2) [reply en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md]: makes sense to me!

**Review by hawc2 — APPROVED** [review]

**Comment by jeffblackadar** [comment]
These changes look great to me, thank you @charlottejmc.  I approve these changes.

I had some unrelated questions/edits which I can make once these changes are merged. I'll mention them in case it's useful now:

 

- [x] 1. "These commercially based services perform more reliably with legible handwriting in a standardized presentation, such as written on straight lines."

Jeff: I was wondering about "such as written on straight lines." should it be "such as being written on straight lines." (I defer to you)


- [x] 2. "Running the cell will store the key and endpoint URL as environment variables within Python's os library, which will allow Python to use these values to communicate with Azure. The expected printed result is this:"

Jeff: As the environment variables aren't stored in the library, can we say "as environment variables in memory using Python's os library...."


- [ ]  3. "This section will allow you to transcribe handwriting from an image found online, which requires the image's URL. For this example, we'll use https://github.com/programminghistorian/ph-submissions/tree/gh-pages/assets/transcribe-handwritten-text-with-python-and-azure/td_00044_b2.jpg1."

Jeff: This link is 404.


- [ ]  4. "Here is an example image to download."

Jeff: This one is also 404.


- [x]  5. "This section will allow you to transcribe handwriting of an image stored in your Python environment in the same as in Project 2."

Jeff: add way?  "This section will allow you to transcribe handwriting of an image stored in your Python environment in the same way as in Project 2."


- [x]  6. In the Summary it has "With this code, you can add more steps to process multiple images and store the transcribed text in a file or database.  Using Python, a loop can transcribe all of the images in a directory or on a series of web pages."

Jeff:  Now that we have Project 3 we should edit this to:  "In Project 3 you added steps to process multiple images and store the transcribed text in a file."


- [x]  7.  My bio - just in case! - Jeff Blackadar has a Master of Arts in History with a specialization in Data Science from Carleton University.


Thank you for all of the work on this so far!  Jeff

**Comment by jeffblackadar** [comment]
Just to note I approve this, but believe it needs to stay open.

**Review by jeffblackadar — CHANGES_REQUESTED** [review]
I've commented on a few items for your consideration before merging please.

**Comment by charlottejmc** [comment]
Thank you @jeffblackadar for your reply. 

I have committed the changes you mentioned to the copyedit branch already. I made a slight adjustment here: "In Projects 3 and 4, you added steps to process multiple images and store the transcribed text in a file."

The links you mention currently show an "error 404" because we have changed the links to relative links, in preparation of moving the linked files over to our Jekyll repository for publication. This will resolve itself once the lesson is published! 

Thank you for your updated bio! We will add this in when the lesson is ready to be published.

> I've commented on a few items for your consideration before merging please.

Does this mean you have also made direct feedback on my edits in the files? I do think you are simply referring to the list of comments you've posted above, but I wanted to double-check with you.

**Comment by jeffblackadar** [comment]
Hi @charlottejmc,
Q: Does this mean you have also made direct feedback on my edits in the
files?
A: I did not make direct feedback - I didn't know how to do it and also did
not want to mess up copy editing.  I figured I would make comments to the
side and they could be looked at.

A question I had was I wanted to see the lesson in natural web page mode
with the images. I forget how to open that, could you remind me of the URL
pls?  I was thinking of running the lesson code again just to be sure.
Thanks for this
Jeff

**Comment by anisa-hawes** [comment]
Thank you for the clarification, @jeffblackadar.

Charlotte is on Annual Leave today so I'll merge this PR in, and then the copyedited version will be available to review in the webpage preview. I'll share the link with you via the main Issue.

Thank you for your patience. Anisa

**Pull request merged**
